pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerGauge.java

```diff
@@ -69,6 +69,10 @@
   // Estimated size of offline table
   OFFLINE_TABLE_ESTIMATED_SIZE("OfflineTableEstimatedSize", false),
 
+  // Size of an uploaded offline segment
+  OFFLINE_SEGMENT_SIZE("OfflineSegmentSize", false),
```

Comment:
Let's name it `LAST_PUSHED_SEGMENT_SIZE` to be more specific

Reply:
great suggestion